tools/ogstools/shp2msh.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="shp2msh" name="ogstools shp2msh" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>
+        Convert shp to vtu.
+    </description>
+    <macros>
+        <import>macros.xml</import>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <expand macro="ogsTools_requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+        export SHAPE_RESTORE_SHX=YES &&    
+        ln -s '$input_shp' input.shp &&
+        shp2msh --input input.shp --output output.vtu
+        #if str($cellsize)!="":
+            --cellsize '$cellsize'
+        #end if
+        '$meshing'
+        '$simplify'
+    ]]></command>
+    <inputs>
+        <param name="input_shp" type="data" format="shp" label="shp input data" argument="-i"/>
```

Comment:
I would try to harmonize the input with the msh2vtu tool (label here should also be changed "Input SHP File")